packages/cli/src/cmds/preview/depsChecker/dotnetChecker.ts

```diff
@@ -398,7 +398,7 @@ export class DotnetChecker implements IDepsChecker {
   }
 
   private static getDefaultInstallPath(): string {
-    return path.join(os.homedir(), `.${ConfigFolderName}`, "bin", "dotnet");
+    return path.join(os.homedir(), `.${ConfigFolderName}`, "bin", "dotnet test"); // TODO: fix it after testing
```

Comment:
"dotnet test" => "dotnet"?

Reply:
yes, will be changed after testing.

Reply:
rename test folder pr link: https://github.com/OfficeDev/TeamsFx/pull/1574